components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -765,31 +765,31 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
             // an image served by the target site, the img load is a simple html
             // request and not an AJAX request, thus bypassing the CORS in this
             // case.
-            this._appendLine('Validating if IAP is up and running...');
+            const expectedTimeSecs = 30 * 60; // 30m
             const startTime = new Date().getTime() / 1000;
             const img = document.createElement('img');
-            img.src = dashboardUri + 'assets/favicon-32x32.png' + '?rand=' + Math.random();
+            const iconFilename = this.state.kfversion === Version.V06 ?
+                'favicon-32x32.png' : 'kf-logo_64px.svg';
```

Comment:
If user change version during wait time, we might be watching the wrong path?
Should we still keep pulling both path for v0.5 and v0.6?

Reply:
Ahh good catch. I changed the other references to `this.state` to be captured in the closure. I'll fix this.